src/_product-updates/2025-07-15-documentation-fixes-and-improvements.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Documentation fixes and improvements
+tags: [Improvement]
+---
+Even if our product grows with new features, taking the time to maintain, enhance, and polish existing features is crucial to ensure a solid documentation experience. That's why we always give ourselves time to tackle small enhancements and fixes alongside the development of new features. Here's a list of what's been worked on.
+
+Improvements:
+- You can now add more than one additional link to the navigation bar by using [our new custom extension](https://docs.bump.sh/help/publish-documentation/external-links/) called `x-externalLinks` instead of the `externalDocs` property,
+- When the link to an operation or group no longer exists, it now returns a 404 page instead of redirecting to the top of the documentation, providing a clearer feedback,
+- The global stability of the platform has been improved by optimizing requests and setting up deeper cache strategies. It's an ongoing endeavour that we'll continue for the next few months,
+- The real-time status of the platform can now be followed on [status.bump.sh](https://status.bump.sh),
```

Comment:
This was already the case, but this page has been really improved for a couple of weeks.